README.md

```diff
@@ -1 +1,179 @@
-# audit-service-deploy
\ No newline at end of file
+## Audit Service Deployment
+
+### Introduction
+
+The Audit Service is designed to provide auditing of user and system actions by defining the required events and the information associated with each event. An application audit event definition file can be used to generate an application-specific, client-side auditing library. The Audit service is multi-tenant aware. Applications send events using the generated client-side auditing library to Elasticsearch where they are indexed according to each tenant.
+
+### Deployment Repository
+
+This repository provides the necessary files to easily get started using the Audit Service.
+
+The only pre-requisite required to get started is that Docker must be available on the system.
+
+The deployment files are in Docker Compose v3 format, and they are compatible with both Docker Compose and Docker Stack.
+
+The deployment file, at present, only references Elasticsearch.
+
+### Demonstration
+
+The Docker Compose file contains the following services:
+
+
+<table><img src=./images/audit_service_deploy.jpg /></table>
+
+**Elasticsearch**  
+[Elasticsearch](https://www.elastic.co/products/elasticsearch) is a search engine based on [Lucene](https://lucene.apache.org/core/). It provides a distributed, multi-tenant capable full-text search engine with an HTTP web interface and schema-free JSON documents. Elasticsearch is developed in Java and is released as open source under the terms of the Apache License.
+
+### Usage
+
+1. Download the files from this repository  
+You can clone this repository using Git or else you can simply download the files as a Zip using the following link:  
+https://github.com/CAFAudit/audit-service-deploy/archive/develop.zip
+
+2. Configure the external parameters if required  
+The following parameters may be set:
+
+    <table>
+      <tr>
+		<th>Docker Compose Variable</th>
+        <th>Environment Variable</th>
+        <th>Default</th>
+        <th>Description</th>
+      </tr>
+      <tr>
+        <td>cluster.name</td>
+		<td>ELASTICSEARCH_CLUSTER_NAME</td>
+        <td>docker-cluster</td>
+        <td>A node can only join a cluster when it shares its cluster.name with all the other nodes in the cluster</td>
+      </tr>
+      <tr>
+        <td>ES_JAVA_OPTS</td>
+		<td>ES_JAVA_OPTS</td>
+        <td>-Xmx256m -Xms256m</td>
+        <td>Environment variable to set heap size <br />e.g. to use 1GB use ES_JAVA_OPTS="-Xms1g -Xmx1g"</td>
+      </tr>
+      <tr>
+        <td>ports</td>
+		<td>ELASTICSEARCH_HTTP_PORT</td>
+        <td>9200</td>
+        <td>HTTP Port used for RESTful API</td>
+      </tr>
+      <tr>
+        <td>ports</td>
+		<td>ELASTICSEARCH_NETWORK_PORT</td>
+        <td>9300</td>
+        <td>Network Communication Port used for Java API, the Elasticsearch transport protocol and Cluster Communications</td>
+      </tr>
+    </table>
+
+3. Disable X-Pack  
```

Comment:
This feels like an internal detail.  I expect the audience here to be somebody trying to test out CAF Auditing.  I don't see why this sort of detail is relevant to them.